### PR TITLE
Include hikaricp in Getting Started dependencies.

### PR DESCRIPTION
**Pull request by stewSquared**
Saves people time figuring out configuration. See #1330

### CODE REVIEW

**Comment by typesafehub-validator** [comment]
Hi @stewSquared,

Thank you for your contribution! We really value the time you've taken to put this together.

Before we proceed with reviewing this pull request, please sign the Typesafe Contributors License Agreement:

[http://www.lightbend.com/contribute/cla](http://www.lightbend.com/contribute/cla)

File: slick/src/sphinx/gettingstarted.rst [hunk 1]
```diff
@@ -33,7 +33,8 @@ following to your build definition - ``build.sbt`` or ``project/Build.scala``:
 .. parsed-literal::
   libraryDependencies ++= Seq(
     "com.typesafe.slick" %% "slick" % "|release|",
-    "org.slf4j" % "slf4j-nop" % "1.6.4"
+    "org.slf4j" % "slf4j-nop" % "1.6.4",
+    "com.typesafe.slick" %% "slick-hikaricp" % "3.1.1"
```
Comment (tminglei) [inline slick/src/sphinx/gettingstarted.rst]: It's better to change the `"3.1.1"` to `"|release|"`.

Comment (cvogt) [inline slick/src/sphinx/gettingstarted.rst]: What I was thinking, newbies have fewer problems if they add hikaricp unnecessarily compared to when they don't add it when it is needed. So putting it there defensively is probably a good thing. Also, trying to remember if sphinx supports variables that inject the current version ... not sure

File: slick/src/sphinx/gettingstarted.rst [hunk 2]
```diff
@@ -61,9 +62,10 @@ log output.
 The Reactive Streams API is pulled in automatically as a transitive dependency.
 
 If you want to use Slick's connection pool support for HikariCP_, you need to add
-the ``slick-hikaricp`` module as a dependency in the same way as shown for ``slick``
-above. It will automatically provide a compatible version of HikariCP as a transitive
-dependency.
+the ``slick-hikaricp`` module as a dependency in the same way as shown above. It
```
Comment (d6y) [inline slick/src/sphinx/gettingstarted.rst]: The phrase "in the same way" doesn't add anything here. I suggest cutting it so the line can read: "as a dependency, as shown above".

File: slick/src/sphinx/gettingstarted.rst (hunk 1 above)
Comment (d6y) [inline slick/src/sphinx/gettingstarted.rst]: Just tried it locally to check the version injection in sphinx, and I get `"com.typesafe.slick" %% "slick" % "3.2.0-SNAPSHOT"`.   Using `|release|` in the hikaricp line would be good!

If users are getting caught out on this one I think it's better to include the dependency it by default that not.  An reasonable alternative would be to include the dependency line as an example in the text where pooling is mentioned... but one way or another it should be here.

**Comment by stewSquared** [comment]
@d6y I completely agree -- go for it. I was just adding words to the existing text.

File: slick/src/sphinx/gettingstarted.rst (hunk 1 above)
Comment (cvogt) [inline slick/src/sphinx/gettingstarted.rst]: ok, looks like the version number should be fixed before the merge then.

**Comment by d6y** [comment]
@stewSquared are you able to update this PR to use `|release|` version (see https://github.com/slick/slick/pull/1466#discussion_r57107109) and make that small text adjustment ... so we can merge?

**Comment by stewSquared** [comment]
@d6y All done! Thanks for your patience.

**Comment by cvogt** [comment]
LGTM

**Comment by tminglei** [comment]
Merged. _Thanks! :-)_

**Pull request merged**